spec/create_table_statement_spec.cr

```diff
@@ -24,4 +24,90 @@ describe LuckyMigrator::CreateTableStatement do
       email text)
     SQL
   end
+
+  describe "indices" do
+    it "can create tables with indices" do
+      statement = LuckyMigrator::CreateTableStatement.new(:users).build do
+        add name : String, index: true
+        add age : Int32, unique: true
+        add email : String
+
+        add_index :email, unique: true
+      end
+
+      statement.should eq <<-SQL
+      CREATE TABLE users (
+        id serial PRIMARY KEY,
+        created_at timestamp NOT NULL,
+        updated_at timestamp NOT NULL,
+        name text NOT NULL,
+        age int NOT NULL,
+        email text NOT NULL)
+        CREATE INDEX users_name_index ON users USING btree (name);
+        CREATE UNIQUE INDEX users_age_index ON users USING btree (age);
+        CREATE UNIQUE INDEX users_email_index ON users USING btree (email);
+      SQL
+    end
+
+    it "defaults to btree" do
```

Comment:
I'm not sure this test is necessary since the test above tests this as well. I think it could be removed and we'd still have good coverage

Reply:
Yeah true, I'll remove it